# Gaussian Mixture Model message
Header header

# Number of components in the mixture
int32 n_components

# Array of Gaussian components
GaussianComponent[] components